### DISCUSSION TITLE
新版本ui改成灰色，是统一的吗，根本分不清，眼睛要瞎了，并且为什么更新不了呢，关卡选择时选不了活动关

### DISCUSSION THREAD

**Discussion by 714938** [MaaAssistantArknights/MaaAssistantArknights#7274] [Q&A] [answered]
![QQ截图20231112031757](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/150550307/56bee9ac-d4b9-4dd1-840e-cffa519f41a9)
![QQ截图20231112031644](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/150550307/b97bbd38-961d-4402-b2c2-94e17a1438bd)

**Comment by ABA2396** [comment, marked as answer]
wpf应用的默认颜色是跟着你的Windows主题颜色走的
